kvm-guests/common/lxc/iperf3.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+#
+# requires:
+#  bash
+#
+set -e
+set -o pipefail
+
+chroot_dir=${1}
+lxc_suffix=${2}
+file=iperf3-3.0.10-1.el6.x86_64.rpm
```

Comment:
Specified version looks bad.
EPEL yum repositry's policy is removing old version.
if iperf package released, that version package will be removed.